### ISSUE TITLE
Add information for interested speakers

### ISSUE THREAD

**Issue by Ephem** [stockholm-react-js/governance#2]
We should set up a place where people interested in speaking can propose talks. It would be nice to have this up pretty quickly, so maybe we should go with a Google form at least to start with (and maybe as a permanent solution) to not be blocked by the website. Lets make it really friendly and welcoming. :)

We should also add some information directed at potential speakers.

**Comment by Pallinder** [comment]
Couldn't we consolidate all these things on the meetups repo? So we have meetups looking for speakers and speakers looking for meetups in the same place. What do you think?

**Comment by Ephem** [comment]
One doesnt exclude the other I think. What I was thinking when I wrote this issue was that many speakers might not be comfortable suggesting talks in the open?

Not at all sure about a good solution for that though. :)
